appengine/endpoints-frameworks-v2/backend/README.md

```diff
@@ -28,11 +28,11 @@ To build the project:
 To generate the required configuration file `swagger.json`:
 
 0. Download and unzip the [Endpoints Framework tools
-package](http://search.maven.org/remotecontent?filepath=com/google/endpoints/endpoints-framework-tools/2.0.0-beta.4/endpoints-framework-tools-2.0.0-beta.4.zip).
+package](http://search.maven.org/remotecontent?filepath=com/google/endpoints/endpoints-framework-tools/2.0.0-beta.7/endpoints-framework-tools-2.0.0-beta.7.zip).
 
```

Comment:
I'll ok this if you insist, however:

1. You could do this from maven and then add a a -Pproperty or -Dflag that causes maven to execute this.  There are examples of this [runlocal](https://github.com/GoogleCloudPlatform/cloud-bigtable-examples/blob/master/java/dataflow-connector-examples/pom.xml#L124-L151) or [HelloWorldWrite](https://github.com/GoogleCloudPlatform/cloud-bigtable-examples/blob/master/java/dataflow-connector-examples/pom.xml#L153-L183)  that would be a better experience for first time users.

It would also be a bit more sustainable.